test/unit/TPML-marshal.c

```diff
@@ -43,20 +42,18 @@ tpml_marshal_success(void **state)
     sel.pcrSelections[1].pcrSelect[0] = 0xdd;
     sel.pcrSelections[1].pcrSelect[1] = 0xee;
 
-    ptr2 = (TPML_PCR_SELECTION *)(buffer);
-
     rc = Tss2_MU_TPML_PCR_SELECTION_Marshal(&sel, buffer, buffer_size, NULL);
-    assert_int_equal (rc, TSS2_RC_SUCCESS);
```

Comment:
We are skipping the check of rc after marshaling.

Reply:
Ups, that was our bad... Remove one too many lines...
Will update...